org.ppsspp.PPSSPP.yml

```diff
@@ -51,3 +55,26 @@ modules:
 
       - type: file
         path: org.ppsspp.PPSSPP.metainfo.xml
+
+  - name: ppsspp-localization
```

Comment:
Do you make sure that ppsspp's local files satisfy the local extension?  
Are there any doc which describes the requirements?

Reply:
> Do you make sure that ppsspp's local files satisfy the local extension?

All I can say is that it works. You can see it for yourself by installing the CI build. :wink:

> Are there any doc which describes the requirements?

No, this is the way PPSSPP handles l10n files during runtime. It just stores all l10n in the `asstes/lang` directory and expects to find them there during runtime.

Reply:
Yes, that’s what I have said. I have tested it. You can do it too. What else do you want? :confused:

Reply:
i think there should be some doc or references.  
Otherwise, this is like black box magic.

Reply:
There are no external documents for this. There is just source code. PPSSPP uses a proprietary scheme and framework for l10n.

Reply:
https://github.com/flatpak/flatpak/blob/1.14.0/common/flatpak-dir.c#L15388

Reply:
I am not sure what you are trying to say by this?

Reply:
>        locale-subset (boolean)
>         If set, then the extensions are partially downloaded by default, based on the currently
>         configured locales. This means that the extension contents should be a set of directories  
>         with the language code as name. Available since 0.9.13 (and 0.6.6 for any extensions
>         called *.Locale)

I want to make sure flatpak can download correct l10n file for different language config.

Reply:
It does. :confused: I do not understand why you simply do not do a test install?

Reply:
I need to explain to me why it works, rather than it works.  
Otherwise, I can't modify anything to fix problems.

Reply:
Oh, I think I might understand what you mean. Have a look at [Flatpak 0.9.98](https://github.com/flatpak/flatpak/releases/tag/0.9.98) release notes:
> * Added new extension property `locale-subset`, which makes the extension point act like a locale extension (i.e. only install the subset configured by the locale).

Basically, this feature was targeted since day one of Flatpak’s development. Besides, there should be a Flatpak feature list somewhere…